surrealdb/surrealdb_test.go

```diff
@@ -0,0 +1,121 @@
+package surrealdb
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/stretchr/testify/require"
+	"os"
+	"testing"
+	"time"
```

Comment:
Tests should use `testcontainers-go`. See some of the other packages

Reply:
Reference: https://testcontainers.com/modules/surrealdb/?language=go